cmd/ebrelayer/types/types.go

```diff
@@ -40,16 +40,16 @@ func (d Event) String() string {
 
 // EthereumEvent struct is used by LogLock and LogBurn
 type EthereumEvent struct {
-	EthereumChainID       *big.Int
-	BridgeContractAddress common.Address
-	ID                    [32]byte
-	From                  common.Address
 	To                    []byte
-	Token                 common.Address
 	Symbol                string
+	EthereumChainID       *big.Int
 	Value                 *big.Int
 	Nonce                 *big.Int
 	ClaimType             ethbridge.ClaimType
+	ID                    [32]byte
```

Comment:
We should not be changing the order in which items in our structs are declared if we can help it. It seems like this change doesn't really even change the keys or values or their types, instead it just moves things around. We should revert these changes.

Reply:
Non issue. Closing.